control-plane/tests/test_service_persistence_action_reconciliation_reviewed_requests.py

```
@@ -0,0 +1,872 @@
+        recommendation = service.record_case_recommendation(
+            case_id=promoted_case.case_id,
+            review_owner="analyst-001",
+            intended_outcome="Review repository owner change evidence before any approval-bound response.",
+            lead_id=service.record_case_lead(
+                case_id=promoted_case.case_id,
+                triage_owner="analyst-001",
+                triage_rationale="Privilege-impacting change needs durable business-hours follow-up.",
+                observation_id=observation.observation_id,
+            ).lead_id,
+        )
+        expires_at_utc = datetime.now(timezone.utc) + timedelta(hours=4)
```

Ruff (0.15.10)

[error] Undefined name `datetime`

(F821)